src/awscdk/awscdk-app-ts.ts

```diff
@@ -211,7 +211,7 @@ export class AwsCdkTypeScriptApp extends TypeScriptAppProject {
     switch (this.package.packageManager) {
       case NodePackageManager.BUN:
         // bun doesn't support tsconfig overrides yet
-        return `bunx ${appEntrypoint}`;
+        return `bun ${appEntrypoint}`;
```

Comment:
I think the explicit `bun run` would be preferred:

```suggestion
        return `bun run ${appEntrypoint}`;
```

Reply:
updated